designs/Transitive-Dependencies.md

```diff
@@ -0,0 +1,150 @@
+
+# Transitive Dependencies
+
+* Status: Reviewed
+* Author(s): [Jon Douglas](https://github.com/NuGet)
```

Comment:
Note--this is a NuGet link. Is that intentional?

Reply:
Copy / pasted a template and never updated it.

Reply:
Updating via commit